infra/src/main/scala/fr/gospeak/infra/services/storage/sql/ProposalRepoSql.scala

```diff
@@ -162,11 +162,11 @@ object ProposalRepoSql {
     (buildSelect(tableFr, fieldsFr, page.all).query[Proposal], buildSelect(tableFr, fr0"count(*)", page.where).query[Long])
   }
 
-  private[sql] def selectPage(speaker: User.Id, status: Proposal.Status, params: Page.Params): (doobie.Query0[(Cfp, Proposal)], doobie.Query0[Long]) = {
-    val selectedTables = Fragment.const0(s"$table p INNER JOIN ${CfpRepoSql.table} c ON p.cfp_id=c.id")
-    val selectedFields = Fragment.const0((CfpRepoSql.fields.map("c." + _) ++ fields.map("p." + _)).mkString(", "))
-    val page = paginate(params, searchFields, defaultSort, Some(fr0"WHERE p.status=$status AND p.speakers LIKE ${"%" + speaker.value + "%"} "), Some("p"))
-    (buildSelect(selectedTables, selectedFields, page.all).query[(Cfp, Proposal)], buildSelect(selectedTables, fr0"count(*)", page.where).query[Long])
+  private[sql] def selectPageWithEvent(speaker: User.Id, params: Page.Params): (doobie.Query0[(Event, Proposal)], doobie.Query0[Long]) = {
```

Comment:
why not keeping the previous method ? And having a `status` parameter can be helpful in the future